### ISSUE TITLE
Issue with the payment for API subscription

### ISSUE THREAD

**Issue by jmgb** [janlukasschroeder/stock-market-news-api-nodejs#1]
I am trying to buy an API license, but there is an issue with the payment button (it does not work). I sent several emails to support@newsfilter.io , but no one replied. 
Can you help me? 
 
![Captura de pantalla 2024-05-15 214824](https://github.com/janlukasschroeder/stock-market-news-api-nodejs/assets/4900807/d378b2ea-0a9c-4279-ae50-48db008478a9)

**Comment by lcfPaisaDouble** [comment]
Hello, we are facing the same issue with payment. Were you able to figure it out? Or any other solution/alternative for similar service/product?

Much thanks in advance.

**Comment by jmgb** [comment]
hello! the developer never replied, so I found another similar service:
https://www.stocktitan.net/news/live.html

**Comment by lcfPaisaDouble** [comment]
Thanks @jmgb 
Do you know if stock titan provides api support? I could not find it on their website.

Do you know any other news sources that provide real time new feeds via api?

Thank you

**Comment by BentlyM** [comment]
> Thanks [@jmgb](https://github.com/jmgb) Do you know if stock titan provides api support? I could not find it on their website.
> 
> Do you know any other news sources that provide real time new feeds via api?
> 
> Thank you

Yep, that’s actually why I built https://www.gridnews.io/ — it’s a WebSocket-based news API made for devs. Streams articles for stocks, crypto, AI, and more. Might be what you're looking for.
